scripts/building/promptflow-tracing/dev-setup.py

```diff
@@ -0,0 +1,118 @@
+# ---------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# ---------------------------------------------------------
+
+import json
```

Comment:
I we have multi files fo rdifferent project, we should able to share some utils?

Reply:
have updated the script to work more like common one.